src/controllers/connectionConfig.ts

```diff
@@ -0,0 +1,67 @@
+import { rejects } from "assert";
+import { resolve } from "path";
+
+const Connection = require('tedious').Connection;
+const RequestStatement = require('tedious').Request;
+let connection;
+
+export function setupConnection(portVal){
+    const config = {

```

Comment:
It probably makes sense to pull this out of this method - we don't ever expect it to change, and by putting it here we have to create this variable and assign it every single time the method is called (as opposed to just once on setup if it was outside the method).